cmd/controller/app/options/options.go

```diff
@@ -97,7 +98,7 @@ const (
 )
 
 var (
-	defaultACMEHTTP01SolverImage                 = fmt.Sprintf("quay.io/jetstack/cert-manager-acmesolver:%s", util.AppVersion)
+	defaultACMEHTTP01SolverImage                 = fmt.Sprintf("quay.io/jetstack/cert-manager-acmesolver-%s:%s", runtime.GOARCH, util.AppVersion)
```

Comment:
This will actually break the amd64 case, as in order to maintain backwards compatibility we *don't* push an `amd64` suffixed version of the images.

We'll need a special case here to handle this - perhaps break this out into a method:

```
func computeACMEHTTP01SolverImage(arch string) string {
    if arch == "amd64" { ... }
}
```

Reply:
Updated now. Is it OK to have that method directly here? I was wondering if I should put it somewhere under `pkg/util` instead...?